src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/mercury/NavigationSideComponentObject.java

```diff
@@ -27,7 +27,7 @@
   @FindBy(css = ".local-wikia-search a")
   private WebElement searchSuggestion;
 
-  @FindBy(xpath = "//ul[contains(@class, \"local-nav-menu\")]/li/*")
```

Comment:
@tomasznapieralski could you please explain in more details why the @justnpT's solution is not appropriate here?
It looks legit for me:
![testing](http://content.screencast.com/users/DmytroRets/folders/Jing/media/133e1934-1e90-4e96-992d-fe5333dcf6cc/00000396.png)

Only one note which came to my mind: 
if you really want to transform xpath to css really identically you should use
`ul[class*='local-nav-menu'] li > *`